horace_core/sqw/file_io/@sqw_binfile_common/sqw_binfile_common.m

```diff
@@ -175,6 +175,8 @@
         [sqw_data,obj] = get_data(obj,varargin);
         % read pixels information
         pix    = get_pix(obj,varargin);
+        pix = get_pix_at_indices(obj, pix_starts, pix_ends);
+        pix = get_pix_in_ranges(obj, pix_starts, pix_ends);
```

Comment:
Not clear that the second pix assignment doesn't overwrite the first. Presumably for this to work pix is a reference to the pixel data in obj (?) in which case the comment might be better as "get ref to pixels info"

Reply:
These are just method declarations